docs/team/prachi2023.md

```diff
@@ -8,17 +8,61 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
 
 *Roles (short for ‘in charge of role’): Description*
 
-- 
+- Documentation: responsible for the parts regarding saving and loading of files 
+- Implementation of I/O
+- In charge of `pin-n`, `view-n`
 
 ## Summary of Contributions
 
 ### Features implemented
+  1. Completed the following features and test code
+     - `pin-n`
+     - `view-n`
+        
+  1. Ensure proper loading of previously saved data 
+  1. Ensure that all the data is saved when changes are made, specifically in the following classes
+     - `AddEventCommand`
+     - `AddNoteCommand`
+     - `ArchiveNoteCommand`
+     - `UnarchiveNoteCommand`
+     - `CreateTagCommand`
+     - `DeleteEventCommand`
+     - `DeleteNoteCommand`
+     - `EditEventCommand`
+     - `EditNoteCommand`
+     - `PinCommand`
+     - `TagEventCommand`
+     - `TagNoteCommand`
+   
+   1. Saving of all the notes and events when the programme ends 
```

Comment:
Need to elaborate on your functions. Look at my PPP or Jun Xiang's to get an idea (PPP or PR depending on what time you see it).